File: Python/D104.py
import math


s=input()
a = int(s.split(' ')[0])
b = int(s.split(' ')[1])
c = int(s.split(' ')[2])

delta = b**2 - 4*a*c

if delta < 0:
    print("None")
elif delta == 0:
    x = (-b + math.sqrt(delta)) / (2*a)  
    print(format(x, ".3f"))
else:
    x = (-b + math.sqrt(delta)) / (2*a)  
    y = (-b - math.sqrt(delta)) / (2*a)  
    if x < y:
        print(format(x, ".3f"))
        print(format(y, ".3f"))
    else:
        print(format(y, ".3f"))
        print(format(x, ".3f"))

File: Java/D104.java
import java.util.Scanner;

public class program {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        if (scan.hasNextLine()) {
            int a = scan.nextInt();
            int b = scan.nextInt();
            int c = scan.nextInt();

            double delta = b*b-4*a*c;
            double x, y;

            if (delta < 0) {
                System.out.println("None");
            } else if (delta == 0) {
                x = (-b + Math.sqrt(delta)) / (2*a);
                System.out.println(String.format("%.3f", x));
            } else {
                x = (-b + Math.sqrt(delta)) / (2*a);
                y = (-b - Math.sqrt(delta)) / (2*a);
                if (x < y) {
                    System.out.println(String.format("%.3f", x) + " " + String.format("%.3f", y));
                } else {
                    System.out.println(String.format("%.3f", y) + " " + String.format("%.3f", x));
                }
            }
        }
    }
}
